frontend/src/routes/Playlist.tsx

```diff
@@ -123,30 +124,15 @@ const PlaylistPage: React.FC<PlaylistPageProps> = ({ playlistFrag }) => {
         return unreachable();
     }
 
-    const items = playlist.entries.map(entry => {
-        if (entry.__typename === "AuthorizedEvent") {
-            const out = readInlineData<VideoListEventData$key>(videoListEventFragment, entry);
-            return out;
-        } else if (entry.__typename === "Missing") {
-            return "missing";
-        } else if (entry.__typename === "NotAllowed") {
-            return "unauthorized";
-        } else {
-            return unreachable();
-        }
-    });
-
     return <div css={{ display: "flex", flexDirection: "column" }}>
-        <Breadcrumbs path={[]} tail={playlist.title} />
-        <PageTitle title={playlist.title} />
+        <Breadcrumbs path={[]} tail={playlist.title ?? ""} />
+        <PageTitle title={playlist.title ?? ""} />
```

Comment:
Reminder to self: why the heck is `title` optional now...

Reply:
Might actually be a relay type generation bug or something. Lets actually ignore it. Maybe add a comment there that this should never be `undefined`